frontends/web/src/routes/device/waiting.tsx

```diff
@@ -64,8 +65,24 @@ export const Waiting: FunctionComponent = () => {
             </div>
             <Guide>
                 <Entry entry={t('guide.waiting.welcome')} shown={true} />
-                <Entry entry={t('guide.waiting.getDevice')} />
-                <Entry entry={t('guide.waiting.lostDevice')} />
+                <Entry entry={{
+                    link: {
+                        text: t('guide.waiting.getDevice.link.text'),
+                        url: 'https://shiftcrypto.shop/',
+                    },
+                    text: t('guide.waiting.getDevice.text'),
+                    title: t('guide.waiting.getDevice.title'),
+                }} />
+                <Entry entry={{
+                    link: {
+                        text: t('guide.waiting.lostDevice.link.text'),
+                        url: (i18n.language === 'de')
+                            ? 'https://shiftcrypto.support/help/de-de/5-backup/8-wie-kann-ich-ein-bitbox02-wallet-in-ein-drittanbieter-wallet-importieren'
```

Comment:
interestingly, you can also use `de-de` with the english suffix :thinking: 

https://shiftcrypto.support/help/de-de/5-backup/8-how-do-i-restore-my-wallet-if-my-bitbox02-is-lost

Reply:
yeah, the url you pasted url should not exist unfortunatelly that tool has no consisten links and as far as i know it uses 2 different links for the same page.

you can also mix or omit completely or write nonsense https://shiftcrypto.support/help/de-de/5-wagagagagaga/8-biiiiiitttttccccoooiiiin 🙈